assets/ajax-cart.js.liquid

```diff
@@ -0,0 +1,627 @@
+      $('input[type="number"]', $cartContainer).each(function() {
+        var el = $(this),
+            currentQty = el.val();
+
+        var itemAdd = currentQty + 1,
+            itemMinus = currentQty - 1,
+            itemQty = currentQty;
+
+        var source   = $("#AjaxQty").html(),
+            template = Handlebars.compile(source),
+            data = {
+              id: el.data('id'),
+              itemQty: itemQty,
+              itemAdd: itemAdd,
+              itemMinus: itemMinus
+            };
+
+        // Append new quantity selector then remove original
+        el.after(template(data)).remove();
+      });
+    }
+
+    // If there is a regular link to remove an item, add attributes needed for ajax
+    if ($('a[href^="/cart/change"]', $cartContainer).length) {
+      $('a[href^="/cart/change"]', $cartContainer).each(function() {
+        var el = $(this).addClass('ajaxcart__remove');
+      });
+    }
+  };
+
+  qtySelectors = function() {
+    // Change number inputs to JS ones, similar to ajax cart but without API integration.
+    // Make sure to add the existing name and id to the new input element
+    var numInputs = $('input[type="number"]');
+
+    if (numInputs.length) {
+      numInputs.each(function() {
+        var el = $(this),
+            currentQty = el.val(),
+            inputName = el.attr('name'),
+            inputId = el.attr('id');
+
+        var itemAdd = currentQty + 1,
+            itemMinus = currentQty - 1,
+            itemQty = currentQty;
+
+        var source   = $("#JsQty").html(),
+            template = Handlebars.compile(source),
+            data = {
+              id: el.data('id'),
+              itemQty: itemQty,
+              itemAdd: itemAdd,
+              itemMinus: itemMinus,
+              inputName: inputName,
+              inputId: inputId
+            };
+
+        // Append new quantity selector then remove original
+        el.after(template(data)).remove();
+      });
+
+      // Setup listeners to add/subtract from the input
+      $('.js-qty__adjust').on('click', function() {
+        var el = $(this),
+            id = el.data('id'),
+            qtySelector = el.siblings('.js-qty__num'),
+            qty = parseInt(qtySelector.val().replace(/\D/g, ''));
+
+        var qty = validateQty(qty);
+
+        // Add or subtract from the current quantity
+        if (el.hasClass('js-qty__adjust--plus')) {
+          qty = qty + 1;
+        } else {
+          qty = qty - 1;
+          if (qty <= 1) qty = 1;
+        }
+
+        // Update the input's number
+        qtySelector.val(qty);
+      });
+    }
+  };
+
+  validateQty = function (qty) {
+    if((parseFloat(qty) == parseInt(qty)) && !isNaN(qty)) {
+      // We have a valid number!
+    } else {
+      // Not a number. Default to 1.
+      qty = 1;
+    }
+    return qty;
+  };
+
+  module = {
+    init: init,
+    load: loadCart
+  };
+
+  return module;
+
+}(ajaxCart || {}, jQuery));
```

Comment:
I assume you did not use any rename functionality (refactoring) provided by your IDE nor used [`git mv`](http://git-scm.com/docs/git-mv) with manual refactoring.
(`git` thinks now that you deleted `ajaxify.js.liquid` and created `ajax-cart.js.liquid` from scratch)
Otherwise this compare would show a nice and clean `git diff`.